gradle/java-module.gradle

```diff
@@ -483,6 +483,29 @@ spotless {
 	}
 }
 
+tasks.check.dependsOn spotlessApply
+
+tasks.register( "ciCheck" ) {
+	group "verification"
+	description "Checks for most CI environments"
+	dependsOn test
+	spotlessApply {
+		enabled = false
+	}
+	spotlessJavaApply {
+		enabled = false
+	}
```

Comment:
Only disabling `spotlessApply` is not enough apparently as `spotlessJavaApply` is run separately (run with `--info` for details).